frontend/packages/component-library-styles/CONTRIBUTING.md

```diff
@@ -0,0 +1,153 @@
+- [📚 Need Help?](#-need-help)
+- [💡 Tips for a Smooth PR Review](#-tips-for-a-smooth-pr-review)
+- [✅ How to Get Your PR Approved Faster](#-how-to-get-your-pr-approved-faster)
+
+## 🛣️ Contribution Roadmap
+
+Here’s a quick guide to the contribution process:
+
+1. **(The most important step!) Create a thread** in the `#ask-design-system` Slack channel about your change.
+
+   - **Get approval** from @markabarrett and/or @moshebaricdo (design team).
+   - For technical details, ask @levadadenys.
+
+2. **Create a separate PR** with `@code-dot-org/component-library-styles` code changes.
+
+   - Make sure the PR follows the [Best Practices](./README.md/#best-practices).
+   - Include relevant updates to tokens, colors, typography, and styles.
+
+3. **Get it approved** by:
+
+   - ✅ @moshebaricdo and/or @markabarrett (design)
+   - ✅ @levadadenys (technical implementation)
+
+4. **Merge the PR** once approved.
+
+5. **Use it in your project → Celebrate → Collect tons of appreciation!** 🎉
+
+## 🎨 How to Add a New Style or Token
+
+Most of the styles come directly from the design system Figma file. Here's a list of what can be possibly added
+and how to do it:
+
+0. Check existing design tokens for duplicates or overlap to prevent redundancy.
+
+1. **Add the token** to the appropriate SCSS file:
+
+   - Fonts → `fonts.scss`
+   - Variables → `variables.scss`
+   - Mixins → `mixins.scss`
+
+2. Ensure the token follows existing naming conventions (e.g., `field-helper-section-white`).
+
+3. If adding a new style:
+
+   - Follow the existing structure and organization.
+
+4. **Test the token or style** across light/dark themes and RTL.
+
+5. Submit a PR following the [Contribution Roadmap](#-contribution-roadmap).
+
+## 🚦Production-ready Checklist:
+
+- Approved by the design team;
+- Defined in the appropriate SCSS file;
+- Tested in light and dark themes;
+- Verified in Storybook (if applicable).
+
+## 🎯 Coding Standards
+
+✅ Use **SCSS** for consistency.  
+✅ Follow the [Best Practices](./README.md#best-practices) section in the README.  
+✅ Use **semantic colors** from `@code-dot-org/component-library-styles/colors.scss` where possible.  
+✅ Keep token names **descriptive** and follow existing naming conventions.  
+✅ Keep file size reasonable — split large files if needed.
+
+## 🧪 How to Test Your Styles
+
+There's currently no automated testing for styles. Here's how you can test your changes manually or through other
+packages automated tests:
+
+1. **Unit Tests(Manually)** — Test RTL(right-to-left) support directly since it's part of Design System scope.
+2. **Storybook Tests** — Ensure Storybook displays the token or style correctly.
+3. **Accessibility Tests** — Run Axe tests to verify accessibility in light and dark themes.
+4. **Visual Regression Tests** — Ensure the style appears correctly across browsers and devices.
+
+➡️ See the full [Testing Guide](./README.md#testing) in the README.
+
+## 🔀 Commit Message Guidelines
```

Comment:
There's a [linter](https://github.com/conventional-changelog/commitlint) for this, so I wonder if we need to add it to the documentation. I'm in favor of automating as much as possible and enforcing through code - documentation tends to get lost.

Reply:
It would be nice to automate it, but right now it's not automated, meaning I don't think we should remove it right now.
Also not sure if we should require every commit to be a conventional one, since in that cases commits like 'review fix', 'small styles updates', etc will be required to be conventional comits and to appear in changelog. Also, what will be with auto github commits like merge staging, commit code review suggestions, etc.
Either way, I think it's worth keeping information in documentation so that people can read more about how everything works.